src/components/molecules/OcTable/OcTable.vue

```diff
@@ -429,6 +447,10 @@ export default {
     background-color: var(--oc-color-background-highlight);
   }
 
+  &-accentuated {
+    background-color: rgba(255, 255, 5, 0.1);
```

Comment:
We'll probably want to introduce a theme-able color (= CSS custom properties (variables)) for this, otherwise it could look pretty bad depending on theming changes by our admin users